src/modules/keyboardmanager/KeyboardManagerEngine/KeyboardManagerEngine.vcxproj

```diff
@@ -34,7 +34,7 @@
     <TargetName>PowerToys.$(MSBuildProjectName)</TargetName>
   </PropertyGroup>
   <PropertyGroup>
-    <OutDir>$(SolutionDir)$(Platform)\$(Configuration)\modules\KeyboardManager\$(ProjectName)\</OutDir>
+    <OutDir>..\..\..\..\$(Platform)\$(Configuration)\$(ProjectName)\</OutDir>
```

Comment:
Just to be sort of consistent everywhere. Some definitely require it because of publish commands treating the csproj as the $(SolutionDir) instead of the .sln. So this needs to be relative to the csproj. Since some require it, we just make them all look the same. for consistency.